test/jdk/java/awt/print/PrinterJob/JobName/PrinterJobName.java

```diff
@@ -21,173 +21,58 @@
  * questions.
  */
 
-/**
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterJob;
+
+/*
  * @test
  * @bug 4205601
  * @summary setJobName should be used by PrinterJob
  * @key printer
+ * @library /test/lib /java/awt/regtesthelpers
+ * @build PassFailJFrame jtreg.SkippedException
  * @run main/manual PrinterJobName
  */
-
-import java.awt.*;
-import java.awt.print.*;
-
 public class PrinterJobName implements Printable {
 
+    static String theName = "Testing the Jobname setting";
```

Comment:
```suggestion
    private static final String theName = "Testing the Jobname setting";
```
Maybe even follow the constant naming convention, `THE_NAME`.